### PR TITLE
#4 'Aims style broken' issue resolved.

### PR DESCRIPTION
**Pull request by SmruthyJoseph**
#4 **'Aims style broken'** issue resolved.

### CODE REVIEW

**Comment by SmruthyJoseph** [comment]
#7 'Aims style broken' issue resolved.

File: wp-content/themes/soraytec/front-page.php [hunk 1]
```diff
@@ -34,8 +34,7 @@
                    
                     $i = 1; $j = 1;
                     while ( $aim_array->have_posts() ) : $aim_array->the_post(); 
-                        $title = get_the_title();
-
```
Comment (AbhijithHaridas) [inline wp-content/themes/soraytec/front-page.php:37]: @SmruthyJoseph what is this for?

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]

File: wp-content/themes/soraytec/front-page.php (hunk 1 above)
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/front-page.php:37]: Sorry, That's not needed. I used the title for giving the aim style before but i changed so now it is not needed.

Comment (AbhijithHaridas) [reply wp-content/themes/soraytec/front-page.php:37]: It's Okay, please remove it

Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/front-page.php:37]: Ok

**Comment by SmruthyJoseph** [comment]
#7 Aims style issue corrected.

**Review by AbhijithHaridas — COMMENTED** [review]
@SmruthyJoseph is this working for you?

**Comment by SmruthyJoseph** [comment]
Yes, It works

File: wp-content/themes/soraytec/front-page.php [hunk 2]
```diff
@@ -31,43 +31,38 @@
...
-                        </div>
-                    <?php elseif( $title == "MISSION" ): ?>  
-                        <!-- Mission -->
+                     <?php endif; ?>
+                     <?php if($i == 2): ?>
                         <div class="vision mission">
+                     <?php endif; ?>
+                     <?php if($i == 3): ?>
+                        <div class="vision values">
+                     <?php endif; ?>
```
Comment (AbhijithHaridas) [inline wp-content/themes/soraytec/front-page.php:46]: @SmruthyJoseph this is bad, find another solution

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]

File: wp-content/themes/soraytec/front-page.php (hunk 2 above)
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/front-page.php:46]: Ok

**Comment by SmruthyJoseph** [comment]
#7 Aims style issues corrected.

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]
Update the Aims listing to be more efficient

**Review by AbhijithHaridas — APPROVED** [review]

**Pull request merged**